os/booting-on-launchtimevps.md

```diff
@@ -0,0 +1,133 @@
+# Running CoreOS on RimuHosting's LaunchtimeVPS service
+
+RimuHosting's LaunchtimeVPS service provides hosted virtual machines.  RimuHosting provide CoreOS as one of their standard images.  Customers can launch these VMs via the web interface or via the RimuHosting server management API.
+
+## About the RimuHosting CoreOS setup
+
+RimuHosting automatically creates the systemd networking files (no need to put networking setup in the cloud init block).
+
+The CoreOS image RimuHosting provide is from the stable channel (updating versions from time to time).
+
+RimuHosting use the CoreOS PXE kernel and initrd (stored on your VM at `/boot`).  We boot up the VM via pv-grub (so the kernel used is provided by CoreOS, not the Xen-based VM host).
+
+### Public SSH keys
+The standard CoreOS image uses the core user (instead of root) and doesn't use a password for authentication. You'll need to add an SSH key(s) via the web interface or add keys/passwords via your cloud-config in order to log in.
+
+Ensure you have set your [SSH public keys][rh-ssh-keys-page].  If you do not have a RimuHosting account, use the register link.
+
+RimuHosting will inject these keys into your instance on setup (no need to modify your cloud-config).
```

Comment:
Not clear if it is or is not neccessary to write keys into cloud-config. This line contradicts line 14 sentence 2.